setup.py

```diff
@@ -41,7 +41,9 @@
         "typing_extensions >= 3.7.4;python_version < '3.8'",
     ],
     entry_points={
-        "console_scripts": ["pyannote-database=pyannote.database.cli:main",],
+        "console_scripts": [
```

Comment:
Looks like change in `setup.py` are unrelated to this PR main topic.
Can you revert those changes?